pages/fellows/Jason-Wong.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: fellow
+pagetype: fellow
+shortname: Jason-Wong
+permalink: /fellows/Jason-Wong.html
+fellow-name: Jason Wong
+title: Jason Wong - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2021-05-19
+  end: 2021-07-24
+photo: /assets/images/team/Jason-Wong.jpg
+institution: University of California, Berkeley
+e-mail: jtwong71@berkeley.edu
+project_title: Developing Lorentz Equivariant Graph Neural Networks for Top Quark Tagging
+project_goal: >
+    Top quark tagging is the process of labeling quarks and gluons from particle accelerator data. Machine learning helps automate this labeling process to help us understand physics and test theories. The goal of this project is to use graph neural networks to improve the accuracy of the labeling process and also imposing Lorentz symmetry into the network architecture to greatly reduce the training time.
+mentors:
+  - Savannah Thais (PU)
+  - Daniel Murnane (UCB)
```

Comment:
```suggestion
  - Daniel Murnane (LBNL)
```
?

Reply:
(Not sure what's preferred here, but using `git grep Daniel Murnane, I see:

```
pages/fellows/JeremyFerguson.md:  - Daniel Murnane ()
pages/fellows/VarunSreenivasan16.md:  - Daniel Murnane (Lawrence Berkeley National Laboratory)
```

The first line is clearly wrong (empty parentheses), but the second one seems reasonable.

Reply:
Thank you, I updated the file using the second option